src/NHibernate/Mapping/SimpleValue.cs

```diff
@@ -1,6 +1,6 @@
 using System;
-using System.Collections;
 using System.Collections.Generic;
+using System.Linq;
```

Comment:
Fix an unexpected use of `Util.EnumerableExtension.First` instead of the Linq one. Those extensions are crutches for the use of non generic `IEnumerable`, which are near from not being used anymore. Mainly need to convert some other `SetParameterList` to a generic form for getting rid of them.